bin/node/src/service.rs

```diff
@@ -417,6 +418,7 @@ pub fn new_authority(
         external_addresses: aleph_config.external_addresses(),
         validator_port: aleph_config.validator_port(),
         protocol_naming,
+        rate_limiter_config: RateLimiterConfig::default(),
```

Comment:
why didnt you pass it via aleph_config?

Reply:
I don't want to make it configurable. Was planning to hide it behind an env variable. What do you think?